src/js/main.js

```diff
@@ -334,6 +343,35 @@ Site.Map = {
 
   },
 
+  showPattern: function() {
+    var _this = this;
+    var patternStyle = parseInt(Math.random() * 2);
+
+    if (patternStyle === 1) {
+      $('#background-pattern-holder').addClass('fill-path');
+    } else {
+      $('#background-pattern-holder').addClass('stroke-path');
+    }
+
+    _this.currentPattern = Math.floor(Math.random() * (_this.patternMax - _this.patternMin + 1)) + _this.patternMin;
+
+    $('svg#pattern-' + _this.currentPattern).addClass('show');
+  },
+
+  changePattern: function() {
+    var _this = this;
+
+    $('.map-pattern').removeClass('show');
```

Comment:
agreed, also i been avoiding jquery where possible, especially for selecting elements. tbh i feel the only thing i appreciate from jquery now a days are the animations functions, like `fadeIn()`. this is just a suggestion.